templates/workspace_services/azureml/terraform/main.tf

```diff
@@ -53,3 +85,14 @@ resource "azurerm_private_endpoint" "mlpe" {
   ]
 
 }
+
+resource "azurerm_application_insights" "ai" {
```

Comment:
A workspace has app insights already. Can't you use it?
If not, just consider that the AI is configured in a simplified approach without the private stuff we use in the workspace level. It might also suffer delete errors.

Reply:
@marrobi how will they access app insights? 
(the current workspace one is used by guacamole for example and it's true we don't want researchers to have access to it).

Reply:
So I ran though this, but struggled to test as the simple method of deploying online endpoints (which uses app insights) doesn't seem to be supported in the secure networking configuration. 

This isn't actually changed in this PR, I just moved the file it is sitting in. Suggest merge the PR as is, and if this comes up as an issue can be addressed separately.

Reply:
ok. will resolve for now.